addons/l10n_ar/models/account_chart_template.py

```diff
@@ -48,18 +45,19 @@ def _load(self, sale_tax_rate, purchase_tax_rate, company):
         Responsibility defined in the company
         """
         self.ensure_one()
-        if company.country_id == self.env.ref('base.ar'):
-            company_responsibility = company.l10n_ar_afip_responsibility_type_id
-            coa_responsibility = self._get_ar_responsibility_match(self.id)
-            if coa_responsibility:
-                company.write({
-                    'l10n_ar_afip_responsibility_type_id': coa_responsibility.id,
-                    'country_id': self.env.ref('base.ar').id,
-                    'tax_calculation_rounding_method': 'round_globally',
-                })
 
+        ar_coa = self.env.ref('l10n_ar.l10nar_base_chart_template') + self.env.ref('l10n_ar.l10nar_ex_chart_template') + self.env.ref('l10n_ar.l10nar_ri_chart_template')
```

Comment:
Why not the regular | ?

Reply:
You could actually do the get_ar_responsibility_match first and see if it matches anything instead of constructing ar_coa

Reply:
> Why not the regular | ?

We do not use the | because we can apply | only for two elements, here we need to match 3 elements: using a | b | c give us syntax error. Thats the reason why we used the + instead of |

> You could actually do the get_ar_responsibility_match first and see if it matches anything instead of constructing ar_coa

Yes, that was the original behavior, directly ask get_ar_responsibility_match, but in the last review with you and Quentin you ask to add to filter by country only for Argentinian country and you add the if "if company.country_id == self.env.ref('base.ar'):". 

I will remove the ar_coa and do directly the _get_ar_responsibility_match()